### ISSUE TITLE
selector-list-comma-newline-after should allow end-of-line comments

### ISSUE THREAD

**Issue by ntwb** [stylelint/stylelint#385]
Right now Stylelint is complaining about:

```css
    a, /* Comment. */
    b { color: pink; }
```

With warning:
```
Expected newline after "," (selector-list-comma-newline-after) [stylelint]
```

**Comment by ntwb** [comment]
Looking at previous issues with similar end-of-line comments #208, #210 I started to create a patch but got no further than docs and tests, the main JS required was out of my league :smirk: 

So here's a diff of the docs and tests I wrote :wink: https://gist.github.com/ntwb/1e9cf07ca622cca3b2c0

**Comment by davidtheclark** [comment]
Thanks for raising this issue. I think I got it fixed.

**Comment by ntwb** [comment]
Awesome, confirming fixed :+1:

**Comment by deepfriedmind** [comment]
This does not seem to be working with `//` comments, e.g.:

```scss
a, // comment
b { color: $pink; }
```

**Comment by ntwb** [comment]
@deepfriedmind Check out the read me for [no-invalid-double-slash-comments](https://github.com/stylelint/stylelint/tree/master/src/rules/no-invalid-double-slash-comments)
> Disallow double-slash comments (//...) which are not supported by CSS

If your issue is using `//` in CSS then you should probably change to using `/* Comment. */` comments or if your using a preprocessor (SASS/SCSS etc) then create a new issue detailing your use case would be the best way forward

**Comment by deepfriedmind** [comment]
@ntwb Yes, this is in reference to SCSS syntax. I've opened #758.

**Comment by ntwb** [comment]
:+1:
